.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: Scala CI
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
would this also enable PR merge blockers?

Reply:
What do you mean? This line just says: "when there is a push to the master branch, run this workflow file here"

Reply:
I recommend to also run tests on pull requests, so this section should look like:
```yml
on:
  pull_request:
  push:
    branches: [ master ]
```